src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
```

Comment:
This code considers that `event` will leak out of the for-loop scope. Though stack overflow says it's a safe construction :)